tests/test_launcher.py

```diff
@@ -64,7 +50,8 @@
 
 
 @pytest.mark.skipif(
-    not get_start_instance(), reason="Skip when start instance is disabled"
+    not get_start_instance() and get_start_instance() is not None,
```

Comment:
what does this mean?

Reply:
I might have overcomplicated things here... 

I think I wanted to make sure that even if ``get_start_instance`` returns ``None``, we don't skip the tests.

I will consider to remove it.